manual/development-process/close-card.md

```diff
@@ -0,0 +1,34 @@
+# Close a story or task
+
+These checklists are intended to help us decide whether we can call a piece of
+work "done".
+
+In both cases, code review can be done by any other member of the team.
+Particularly complex or security-critical code should be reviewed by someone
+with a good understanding of the component being changed. Developers should use
+their judgement to decide whether a task requires a more experienced reviewer.
+
+## Stories
+
+A story is "done" if:
+
+* all code has been reviewed by another developer
+* all the subtasks are merged to the "master" branch
+* all the subtasks have been deployed to production (or staging, if we don't
```

Comment:
Maybe reword: "... (or staging, if we don't have a production environment **available**)"

Reply:
Reworded.